pkg/charm/help.go

```diff
@@ -66,14 +65,14 @@ func helpDesc(heading, body string) {
 	if len(body) > lineWidth {
 		body = FormatParagraph(body, tab, lineWidth)
 	}
-	fmt.Fprint(os.Stderr, hdr+"\n"+body)
+	fmt.Print(hdr+"\n"+body)
```

Comment:
```suggestion
	fmt.Print(hdr + "\n" + body)
```

One more to make the linter happy. 😛